src/pages/landing/ebooks/components/_header-section.tsx

```diff
@@ -71,7 +85,7 @@ const HeaderImage = styled(QueryImage)`
 
     @media ${device.tabletS} {
         width: ${(props) => (props.imgWidth < 600 ? '115%' : '105%')};
-        height: ${(props) => (props.imgWidth < 600 ? '105%' : '105%')};
+        height: ${(props) => (props.imgWidth < 600 ? '115%' : '105%')};
```

Comment:
Was this verified with the design?

Reply:
I think the previous height value was a typo since it had 105% on both condition sides.